### PR TITLE
Food tags & unification

### PR DESCRIPTION
**Pull request by ochotonida**
This adds a few food-related tags (or adds items to existing tags) and modifies a few recipes to increase compatibility a bit.
Full list of changes:

#### Unified items:
- Unified Farmer's Delight's Raw Pasta and Simple Farming's Noodles. I kept Farmer's Delight's pasta because it's edible while Simple Farming's noodles are not.
- Unified Farmer's Delight's Fried Egg and Simple Farming's Cooked Egg. These were already functionally identical, I kept Farmer's Delight's egg because it has a nicer texture.
- Unified Farmer's Delight's and Simple Farming's raw/cooked bacon. Simple Farming's bacon was already disabled a couple of days ago (c2c465c91a3e9c0fe6e47b13d834d0e22c8f9c93) but not properly tagged for some reason, I added tags and updated the recipes.
(I actually [sent a pull request to simple farming](https://github.com/cweckerl/simplefarming/pull/141) some months ago that would have fixed this, but it has yet to be merged😕)

#### New/modified tags:
- I've added Turkey Pieces from Autumnity and Chicken Wings from Simple Farming to the raw/cooked chicken tags, and updated recipes from Simple Farming to use these tags
- Added a Jam tag to make Upgrade Aquatic's mulberry jam an alternative ingredient for Simple Farming's PB&J
- Added a Chocolate tag to allow Create's chocolate to be used in Simple Farming's recipes. I didn't completely unify them (yet?), because of the difference in the amount of hunger/saturation they restore. When going with Create's chocolate, Simple Farming's Candy would restore the same amount of hunger but *less* saturation than it's main ingredient. Using Simple Farming's chocolate on the other hand, automating chocolate with Create might not really be worthwhile anymore. I was thinking of going with Create's and simply doubling the output of the candy recipe, but I thought it'd be better to get someone else's opinion on this first.

#### Miscellaneous changes:
- The burgers from Simple Farming now use Beef Patties instead of Steak
- I've removed Simple Farming's Chicken Wings crafting recipe, and replaced Farmer's Delight's chicken cutting recipe with one that has chicken wings as an additional output
- Removed a redundant recipe removal

### CODE REVIEW

File: kubejs/server_scripts/enigmatica/kubejs/base/recipes/replace_input.js [hunk 1]
```diff
@@ -27,6 +27,14 @@ events.listen('recipes', (event) => {
     event.replaceInput({}, 'pedestals:dustflour', '#forge:dusts/flour');
     event.replaceInput({}, 'create:dough', '#forge:doughs');
     event.replaceInput({}, 'farmersdelight:wheat_dough', '#forge:doughs');
+    event.replaceInput({}, 'create:bar_of_chocolate', '#forge:chocolate_bars');
+    event.replaceInput({}, 'simplefarming:chocolate', '#forge:chocolate_bars');
+    event.replaceInput({}, 'simplefarming:noodles', '#forge:pasta/raw_pasta');
+    event.replaceInput({}, 'simplefarming:jam', '#forge:jams');
+    event.replaceInput({}, 'simplefarming:raw_bacon', '#forge:raw_bacon');
+    event.replaceInput({}, 'simplefarming:cooked_bacon', '#forge:cooked_bacon');
+    event.replaceInput({ mod: 'simplefarming' }, 'minecraft:cooked_chicken', '#forge:raw_chicken');
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/base/recipes/replace_input.js:36]: Are you trying to give people salmonella? :P

Comment (NillerMedDild) [reply kubejs/server_scripts/enigmatica/kubejs/base/recipes/replace_input.js]: I took the liberty of changing `forge:raw_chicken` to `forge:cooked_chicken`, I hope that's what you intended :)

**Review by NillerMedDild — APPROVED** [review]
Very nice stuff! :D

File: kubejs/server_scripts/enigmatica/kubejs/base/recipes/replace_input.js (hunk 1 above)
Comment (ochotonida) [reply kubejs/server_scripts/enigmatica/kubejs/base/recipes/replace_input.js]: Oops, not sure how I messed that up 😅

**Comment by NillerMedDild** [comment]
Oh woops, should've read the description closer.

> I was thinking of going with Create's and simply doubling the output of the candy recipe, but I thought it'd be better to get someone else's opinion on this first.

I think that's a good solution.

**Comment by ochotonida** [comment]
Okay, I'll create another pull request to unify chocolate 👍

**Comment by NillerMedDild** [comment]
Sweet!

**Pull request merged**